uint64 dialogue_id;
uint8 dialogue_result; #0: ok; #1: dialogue failure; #2 system failure;
---
uint8 result;